src/confcom/azext_confcom/oras_proxy.py

```diff
@@ -0,0 +1,126 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import subprocess
+import json
+import platform
+from azext_confcom.errors import eprint
+from azext_confcom.config import ARTIFACT_TYPE
+from azext_confcom.cose_proxy import CoseSignToolProxy
+
+host_os = platform.system()
+machine = platform.machine()
+
+
+# discover if there are policy artifacts associated with the image
+# return their digests in a list if there are some
+def discover(
+    image: str,
+) -> bool:
+    arg_list = ["oras", "discover", image, "-o", "json", "--artifact-type", ARTIFACT_TYPE]
+
+    item = subprocess.run(
```

Comment:
we should make the subprocess run (and the subsequent return code check) a utility function where we just pass in the arg list and return the output

Reply:
the error messaging is different for all of them, so I'll make a utility for the call but not the return code check